java-frontend/src/main/java/org/sonar/java/bytecode/cfg/BytecodeCFG.java

```diff
@@ -69,6 +72,10 @@ public String getExceptionType() {
       return exceptionType;
     }
 
+    public Type getExceptionType(SemanticModel semanticModel) {
```

Comment:
Maybe we can remove the `getExceptionType()` method then and only rely on this one. But I didn't tried.